docs/eclipse/release-notes.md

```diff
@@ -5,18 +5,35 @@ title: Release Notes
 
 Here you will find the latest information about releases of the jSparrow Eclipse plugin.
 
+## 3.2.0
+
+_21.02.2019_
+
+### jSparrow is Fit for Java 11
+
+Since the official End of Life of Java 8 has been reached, we have updated jSparrow to be able to run on Java 11 as well as Java 8.
+
+### New Rule
+
+jSparrow 3.2.0 brings a new rule for [collapsing if statements](/rules/collapse-if-statements.html). 
+The motivation behind this rule is to improve the readability of the code by reducing the number of the nested language constructs. 
+
+### Direct Link to Rule Documentation
+
+In the Select Rules Wizard we have added a link to each rule description  which leads directly to a detailed documentation of that rule on our website.
```

Comment:
Previously we officially spelled the SRW as such in release notes:

[..] the “Select Rules” wizard

Also: there are two white spaces between "description" and "which" (I know, it won't show in markdown anyways..)